src/core/search/indices.cc

```diff
@@ -43,12 +47,15 @@ absl::flat_hash_set<std::string> TokenizeWords(std::string_view text) {
 }
 
 // Split taglist, remove duplicates and convert all to lowercase
-absl::flat_hash_set<string> NormalizeTags(string_view taglist) {
+// TODO: introduce ascii support if needed
```

Comment:
you mean `unicode` no? 

Also, why don't we just reject the query if the tag is unicode and we uncertain of the behaviour?

Reply:
Yes, I mean unicode. Would make sense to reject it...